lib/gems/pending/appliance_console/cli.rb

```diff
@@ -207,6 +218,27 @@ def set_external_db
       config.post_activation
     end
 
+    DATE_REGEXP   = /^(2[0-9]{3})-(0?[1-9]|1[0-2])-(0?[1-9]|[12][0-9]|3[01])/
+    TIME_REGEXP   = /^(0?[0-9]|1[0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])/
+
+    def set_date_time
+      date, time = options[:datetime].split("T")
+      if date.validate(DATE_REGEXP) && time.validate(TIME_REGEXP)
+        say "Configuring date and time"
+        autosync = auto_sync? ? options[:autosync] : false
```

Comment:
This line is not necessary, `options[:autosync]` will always be either `true` or `false` so you can pass it directly.